src/index.html

```diff
@@ -5,8 +5,342 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400;700&family=Roboto:wght@400;500;700&family=Space+Grotesk:wght@700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <title>Moyo header</title>
+    <header class="header">
+      <a href="index.html" class="logo">
+        <img
+          src="images/logo.png"
+          alt="moyo_1"
+          class="logo__img"
+        >
+      </a>
+      <nav class="nav" >
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              data-qa="nav-hover"
+              href="#">Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="card" data-qa="card">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="imac"
+         >
+
+        <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+        <p class="card__code" >Product code: 195434</p>
+
+        <div class="card__raiting">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <span class="card__reviews">Reviews: 5</span>
+        </div>
+
+        <div class="card__price">
+          <span class="card__price-text">Price:</span>
+          <span class="card__price-amount">$2,199</span>
+        </div>
+
+        <a
+           href="#"
+           class="card__link"
+           data-qa="card-hover"
+        >
+          BUY
+        </a>
```

Comment:
Let's fix it everywhere
```suggestion
        <a
          href="#"
          class="card__link"
          data-qa="card-hover"
        >
          BUY
        </a>
```